src/psd2svg/converter/shape.py

```diff
@@ -27,7 +27,7 @@ def _get_vector_stroke(self, layer, target):
 
     def _generate_path(self, vsms, command='C'):
         # Iterator for SVG path constructor.
-        anchors = [p for p in vsms.path if p['selector'] in (1, 2)]
+        anchors = [p for p in vsms.path if p['selector'] in (1, 2, 4, 5)]
```

Comment:
added after seeing `Path data record types` in http://www.adobe.com/devnet-apps/photoshop/fileformatashtml/#50577409_17587

Reply:
LGTM